lib/resource-mapper.js

```diff
@@ -24,6 +25,7 @@ class ResourceMapper {
     this._defaultContentType = defaultContentType
     this._indexFilename = indexFilename
     this._types = { ...types, ...overrideTypes }
+    this._isControlFile = new RegExp(`.(?:${fileSuffixes.join('|')})$`)
```

Comment:
Not the right regex: a) fileSuffixes already has a leading dot, so there are two dots now b) dots need to be escaped in regexes (but, since you are creating the regex from a string, actually needs double backslash '`\\.`).

Reply:
Alternatively, you can just pass the regex itself instead of passing fileSuffixes.

Reply:
That makes it difficult to configure the suffixes though =/

Reply:
Hope the latest commit looks good.